lib/resolveModule.js

```diff
@@ -91,6 +98,8 @@ function resolveAsFile(filepath, extensions = []) {
 }
 
 function resolveModule(filepath, {cwd, resolve: resolvecfg = {}}) {
+  const preserveSymlinks = resolvecfg.preserveSymlinks !== undefined
+    ? !!resolvecfg.preserveSymlinks : PRESERVE_SYMLINKS;
```

Comment:
Just thought that it would be nice to move all the necessary coercions of options (and checking) to the plugin initialisation, so we can keep it at one place. But it doesn't look as an issue at the current step :)